### PR TITLE
Match currency logo to currency

### LINKED ISSUE #483: Changing logo of currency based on currency selected.
image

Can we change the currency icon after changing currency in nav bar

### PR DESCRIPTION
**Pull request by askmeegs**
Fixes #483

### CODE REVIEW

File: src/frontend/templates/header.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
                 {{ if $.show_currency }}
                 <div class="h-controls">
                     <div class="h-control">
-                        <img src="/static/icons/Hipster_CurrencyIcon.svg" alt="" class="icon" />
+                        <span class="currencyLogo"> {{ renderCurrencyLogo $.user_currency}}</span>
```
Comment (bourgeoisor) [inline src/frontend/templates/header.html:45]: Is there a style guide or linting for these HTML templates, out of curiosity?
```suggestion
                        <span class="currencyLogo">{{ renderCurrencyLogo $.user_currency }}</span>
```

File: src/frontend/handlers.go [hunk 2]
```diff
@@ -455,3 +456,19 @@ func cartSize(c []*pb.CartItem) int {
...
 }
+
+func renderCurrencyLogo(currencyCode string) string {
+	logos := map[string]string{}
+	logos["USD"] = "$"
+	logos["CAD"] = "$"
+	logos["JPY"] = "¥"
+	logos["EUR"] = "€"
+	logos["TRY"] = "₺"
+	logos["GBP"] = "£"
```
Comment (bourgeoisor) [inline src/frontend/handlers.go:467]: Thoughts?
```suggestion
	logos := map[string]string{
		"USD": "$",
		"CAD": "$",
		"JPY": "¥",
		"EUR": "€",
		"TRY": "₺",
		"GBP": "£",
	}
```

**Review by bourgeoisor — APPROVED** [review]
LGTM! Two minor suggestions.

File: src/frontend/templates/header.html (hunk 1 above)
Comment (askmeegs) [reply src/frontend/templates/header.html:45]: Not that I know of, I typically just reference the godoc https://golang.org/pkg/html/template/

**Pull request merged**